package.json

```diff
@@ -17,7 +17,9 @@
     "test:routes": "BABEL_DISABLE_CACHE=1 HTTP_PROXY= HTTPS_PROXY= TZ=US/Pacific mocha --env=environments/test.env --recursive --require babel-core/register --require babel-polyfill --compilers js:babel-register ./test/setup.js ./test/routes/**/*",
     "migrate:latest": "knex migrate:latest",
     "migrate:rollback": "knex migrate:rollback",
-    "generate-badges": "node scripts/badges.js"
+    "shonkwrap": "shonkwrap",
+    "generate-badges": "node scripts/badges.js",
+    "poll-repositories": "babel --out-dir=dist src scripts/poller.js --copy-files 2>&1 > /dev/null && node dist/scripts/poller.js"
```

Comment:
This builds and runs the poll script. Also it seems that it's build in the build.sh script.

So in theory on production (after whole app is build) calling ./dist/scripts/poller.js is enough, to skip unnecessary build step on prod? Especially that I'm not sure if dev dependencies are available on prod environment.